src/main/kotlin/io/snyk/plugin/ui/toolwindow/panels/JCEFDescriptionPanel.kt

```diff
@@ -176,6 +180,193 @@ class SuggestionDescriptionPanelFromLS(
             .map { allowedChars.random() }
             .joinToString("")
     }
+
+    private fun getCustomScript(): String {
```

Comment:
This feels like 90% of this could in a follow-up PR be pushed down to be served directly from LS.

Reply:
Agree
VS will need this too.
Also we will have to change syntax back to es5

Reply:
I have created jira ticket for this
https://snyksec.atlassian.net/browse/IDE-655